ci-operator/templates/openshift/installer/cluster-launch-installer-src.yaml

```diff
@@ -255,10 +266,31 @@ objects:
         sshKey: |
           ${SSH_PUB_KEY}
         EOF
+        elif [[ "${CLUSTER_TYPE}" == "azure4" ]]; then
```

Comment:
I would prefer cluster type to be `azure` - you can still mount the other secrets to the `azure` name even if the secret name is different.